background.ts

```diff
@@ -1,4 +1,5 @@
-import { spawnSync } from 'child_process';
+import { spawnSync, spawn as standardSpawn } from 'child_process';
```

Comment:
I don't understand why we need this — it's available as `childProcess.spawn` (it's just re-exported).  The name in particular is confusing; at a minimum just call it as `spawn`?

Reply:
Yeah, that works. In fact I didn't want `spawnSync` there at all.